src/middlewares/user.js

```diff
@@ -0,0 +1,17 @@
+
+class UserMiddleWare {
+  static createUserValidation(req, res, next) {
+    req.checkBody('password', 'please supply a valid password and must be a minimum length of 6 characters')
```

Comment:
Can we use validation for a strong password and not just length?

Reply:
@tijesunimi-peters what do you mean by strong password?

Reply:
Combination of caps, characters and numbers

Reply:
Alright, thanks @tijesunimi-peters , can you recommend a condition for that please

Reply:
Reason being that we need to put in place as much security as possible because we are going to be dealing with financial transactions, 
https://www.thepolyglotdeveloper.com/2015/05/use-regex-to-test-password-strength-in-javascript/ @albert85

Reply:
You could add a condition that checks if the password contains at least one uppercase letter and a number. Requesting for a non-alphanumeric character`(e.g _, &, !, etc)` might be too much hassle for users.

Reply:
I don't think it's too much hassle. It's basic security. The record we are going to be dealing with is too sensitive to ignore basic password strength security. @emmaadesile